packages/browser-agent/README.md

```diff
@@ -1,27 +1,148 @@
-This is the main module that clients would import into their applications.
+[![New Relic Experimental header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Experimental.png)](https://opensource.newrelic.com/oss-category/#new-relic-experimental)
 
-It provides APIs to initialize the agent:
+# New Relic Browser Agent
 
+The New Relic Browser Agent instruments your website and provides observability into the performance and behavior of your application. This NPM Library is an ***in-progress*** implementation of the New Relic Browser Agent, accessible through `NPM`. Please see the [Differences](#differences) and [Features](#features) sections to compare how this library differs from the other offerings of the New Relic Browser Agent.
+
+## Installation
+
+### Using Package Managers
+
+Using [npm](https://npmjs.org):
+
+```bash
+npm install @newrelic/browser-agent
+```
+
+or [yarn](https://yarnpkg.com/)
+
+```bash
+yarn add @newrelic/browser-agent
+```
+
+### Directly in HTML/JS
+
+Using [unpkg](https://unpkg.com/)
+
+```html
+<script src="https://unpkg.com/@newrelic/browser-agent/bundled"></script>
```

Comment:
this pattern has changed in recent commits

##### es6
```html
<script src="https://unpkg.com/@newrelic/browser-agent/bundled/es6"></script>
```
##### es5
```html
<script src="https://unpkg.com/@newrelic/browser-agent/bundled/es5"></script>
```

Reply:
It may also be useful to show an example of using the lib from the bundle

```html
<head>
    <script src="https://unpkg.com/@newrelic/browser-agent/bundled"></script>
    <script>
        const { BrowserAgent } = NRBA;
        const options = {
            // See 'Configuring your application'
        }
        const agent = new BrowserAgent()
        agent.start(options).then(() => {
            console.log("Browser Agent Initialized!")
        })
    </script>
    ... other content
</head>
<body>
    <!--- DOM Content --->
    <script>
        // somewhere in your application
        try {
            ...
        } catch(err) {
            agent.noticeError(err)
        }
    </script>
</body>
```